src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -2270,8 +2292,12 @@ ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
 
         uct_ib_mlx5_devx_check_xgvmi(md, cap_2, dev);
         uct_ib_mlx5_devx_check_mkey_by_name(md, cap_2, dev);
+    } else {
+        cap_2 = NULL;
     }
 
+    uct_ib_mlx5_devx_check_multi_path(md, cap, cap_2, dev);
```

Comment:
why not calling it above on line 2295 when cap_2 is initialized? Then no need to initialize `cap_2` to `NULL`

Reply:
intent is to always call `uct_ib_mlx5_devx_check_multi_path()` so that values can be forced to disabled state if we don't have a `cap_2`.

Reply:
are you suggesting to move that call line 2295 and assume default values in this structure are anyways zero?

Reply:
Ok